ccsrc/lib/cppsim/CMakeLists.txt

```diff
@@ -0,0 +1,305 @@
+cmake_minimum_required(VERSION 3.20 FATAL_ERROR)
+
+project(
+  cppsim
+  VERSION 1.0.0
+  LANGUAGES C CXX)
+
+# Minimum required Python version (used both in this file and in the installed CMake configuration)
+set(CPPSIM_PYTHON_VERSION_MIN 3.7.0)
+
+option(BUILD_TESTING "Build the test suite?" OFF)
+
+if(NOT DEFINED ITERATOR_DEBUG_VALUE)
+  set(ITERATOR_DEBUG_VALUE 0)
+endif()
+option(MSVC_ITERATOR_DEBUG "Define _ITERATOR_DEBUG_LEVEL (defaults to 0, can be set using ITERATOR_DEBUG_VALUE)" OFF)
+
+# ==============================================================================
+
+include(GNUInstallDirs)
+
+set(CPPSIM_INSTALL_BINDIR "${CMAKE_INSTALL_BINDIR}")
+set(CPPSIM_INSTALL_SBINDIR "${CMAKE_INSTALL_SBINDIR}")
+set(CPPSIM_INSTALL_SYSCONFDIR "${CMAKE_INSTALL_SYSCONFDIR}")
+set(CPPSIM_INSTALL_INCLUDEDIR "${CMAKE_INSTALL_INCLUDEDIR}/cppsim")
+set(CPPSIM_INSTALL_DATADIR "${CMAKE_INSTALL_DATADIR}/cppsim")
+set(CPPSIM_INSTALL_LIBDIR "${CMAKE_INSTALL_LIBDIR}/cppsim")
+set(CPPSIM_INSTALL_DOCDIR "${CMAKE_INSTALL_DATADIR}/doc/cppsim")
+set(CPPSIM_INSTALL_CMAKEDIR "${CPPSIM_INSTALL_DATADIR}/cmake")
+set(CPPSIM_INSTALL_3RDPARTYDIR "${CPPSIM_INSTALL_LIBDIR}/third_party")
```

Comment:
Modify those so that they are using the `MQ_INSTALL_XXX` CMake variables.

You may redefined the `CPPSIM_INSTALL_XXX` to be relative to `MQ_INSTALL_3RDPARTYDIR` for example